test/e2e/case18_compliance_api_test.go

```diff
@@ -1962,6 +2051,256 @@ var _ = Describe("Test the compliance events API", Label("compliance-events-api"
 			Expect(err).To(MatchError(ContainSubstring("The compliance event already exists")))
 		})
 	})
+
+	Describe("Test authorization", func() {
+		Describe("Test method Get", func() {
+			It("Should return StatusForbidden when it is empty Authorization", func(ctx context.Context) {
```

Comment:
Shouldn't it be unauthorized when no authentication info is provided?

Reply:
It is empty token.. maybe I should change the title